data/helpers.d/php

```diff
@@ -45,7 +45,27 @@ ynh_add_fpm_config () {
 		chown root: "$finalphpini"
 		ynh_store_file_checksum "$finalphpini"
 	fi
-	ynh_systemd_action --service_name=$fpm_service --action=reload
+
+	# Create a config for a dedicated php-fpm service for the app
+	echo "
+[Unit]
+Description=PHP $phpversion FastCGI Process Manager for $app
+After=network.target
+
+[Service] 
+Type=notify
+PIDFile=/run/php/php${phpversion}-fpm-$app.pid
+ExecStart=/usr/sbin/php-fpm${phpversion} --nodaemonize --fpm-config $finalphpconf --pid /run/php/php${phpversion}-fpm-$app.pid
```

Comment:
On my php-fpm service if I also have these lines:

```
ExecStartPost=-/usr/lib/php/php-fpm-socket-helper install /run/php/php-fpm.sock /etc/php/7.0/fpm/pool.d/www.conf 70
ExecStopPost=-/usr/lib/php/php-fpm-socket-helper remove /run/php/php-fpm.sock /etc/php/7.0/fpm/pool.d/www.conf 70
```

But... I have no idea what is it for

Reply:
On stretch ?

Reply:
Yes

Reply:
Where was this file ? I didn't have one !

Reply:
I don't have it on buster btw.

For example (still stretch):
```
root@ynh-dev-stretch:/ynh-dev# cat /etc/systemd/system/multi-user.target.wants/php7.3-fpm.service 
[Unit]
Description=The PHP 7.3 FastCGI Process Manager
Documentation=man:php-fpm7.3(8)
After=network.target

[Service]
Type=notify
ExecStart=/usr/sbin/php-fpm7.3 --nodaemonize --fpm-config /etc/php/7.3/fpm/php-fpm.conf
ExecStartPost=-/usr/lib/php/php-fpm-socket-helper install /run/php/php-fpm.sock /etc/php/7.3/fpm/pool.d/www.conf 73
ExecStopPost=-/usr/lib/php/php-fpm-socket-helper remove /run/php/php-fpm.sock /etc/php/7.3/fpm/pool.d/www.conf 73
ExecReload=/bin/kill -USR2 $MAINPID

[Install]
WantedBy=multi-user.target
```

Reply:
Would it be "sury" that adds these lines? (Do you install/upgrade php via "sury" on your side?).

Reply:
Damn, I missed that directory multi-user.target.wants !
But I have the same script as before
```
PIDFile=/run/php/php7.0-fpm.pid
ExecStart=/usr/sbin/php-fpm7.0 --nodaemonize --fpm-config /etc/php/7.0/fpm/php-fpm.conf
ExecReload=/bin/kill -USR2 $MAINPID
```

Nope, so sury on my side

Reply:
I don't have sury on my buster server either. Anyway, I still don't know the purpose of these lines

Reply:
php-fpm-socket-helper does not exist on my setup, I guess that close the investigation...

Reply:
Mine: https://paste.yunohost.org/inuqiduril.bash

Reply:
Looks like a default socket to use and update before start and after a stop.
I think we can ignore this in our case.